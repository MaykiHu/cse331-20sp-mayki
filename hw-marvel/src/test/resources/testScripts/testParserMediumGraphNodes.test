# Test the parser on a graph that contains 14 nodes
# with varying edges each.

LoadGraph mediumHeroes mediumSizeSuperheroes.tsv
ListNodes mediumHeroes