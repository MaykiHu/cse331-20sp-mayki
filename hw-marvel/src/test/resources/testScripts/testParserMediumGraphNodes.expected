# Test the parser on a graph that contains 14 nodes
# with varying edges each.

loaded graph mediumHeroes
mediumHeroes contains: ANCIENT ONE BARON MORDO/KARL MOR CLEA DORMAMMU DR. STRANGE/STEPHEN  ETERNITY/ADAM QADMON Ernst-the-Bicycling-Wizard Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard Perkins-the-Magical-Singing-Instructor SURTUR UMAR WONG ZOM